src/domain/entities/TranslatableText.ts

```diff
@@ -16,3 +17,24 @@ export const buildTranslate = (locale: string): TranslateMethod => {
 };
 
 export type TranslateMethod = (string: TranslatableText) => string;
+
+export const importTranslate = <T extends TranslatableText>(item: T, language: string, term: string | undefined): T => {
```

Comment:
For first level functions we normally use `function () {}` instead of using arrow functions

Reply:
Yes, nothing wrong with arrow functions, but just to keep things homogeneous. In-line functions on the side tend to use arrow syntax.